tutorials/install.md

```diff
@@ -60,6 +89,23 @@ Add OSRF packages:
     wget http://packages.osrfoundation.org/gazebo.key -O - | apt-key add -
     sudo apt-add-repository -s "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable $(lsb_release -c -s) main"
 
+Only on Bionic, update the GCC compiler version:
+
+    sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-8 800 --slave /usr/bin/g++ g++ /usr/bin/g++-8 --slave /usr/bin/gcov gcov /usr/bin/gcov-8
+
+#### Windows 10
```

Comment:
I think this Windows section should be deleted, the binary install is above and the source install is below.